### ISSUE TITLE
buttons not detecting clicks in jupyter lab 

### ISSUE THREAD

**Issue by ayoungs** [pyviz/panel#218]
button clicks are not showing their clicks value incrementing in jupyter lab 0.35.4.  It works fine in jupyter notebook.

**Comment by philippjfr** [comment]
Could you check whether it works when you wait a few seconds and render the object a second time? I've noticed some issues with the comm initialization if you import the extension and then immediately display a panel.

**Comment by ayoungs** [comment]
That seems to be exactly it, wait a couple seconds before running the cell, and it seems to work fine.

**Comment by philippjfr** [comment]
Should be fixed now when you upgrade to the lates the latest JLab extension: @pyviz/jupyterlab_pyviz v0.6.5.

**Comment by youngsaj** [comment]
Thanks for the fix!  I'll give this a try!

**Comment by philippjfr** [comment]
Note there have been releases since, it's 0.7.0 now.

**Comment by youngsaj** [comment]
It helps one thing, but not another.  It helps with table selections showing up, but doesn't with DynamicMap.  I give it a couple seconds, rerun, and it starts working as above.

**Comment by philippjfr** [comment]
I haven't had any issues since the update so if you could file a new reproducible example that would be great.

**Comment by ayoungs** [comment]
I found holoviews game_of_life that may produce it, it may be more that it's not originally intended for a notebook, but my thought is I need to be Dashboard or notebook.
I removed the last three lines, and just made a panel from dmap.
I just use tap for update, no periodic update line.
I get  No such comm target registered:  ....
When I tap.  I did a conda update --all 
JupyterLab v0.35.4
Known labextensions:
   app dir: -----
        @pyviz/jupyterlab_pyviz v0.7.0 enabled  ok
